pkg/collector/runner/runner_test.py

```diff
@@ -0,0 +1,11 @@
+# Python module for simulating a simple python check
+
+from checks import AgentCheck
+import time;
+
+class TestCheck(AgentCheck):
+    def check(self, instance):
+        # Busy wait for 100ms
+        current_time = time.time()
+        while (time.time() < current_time+0.1):
```

Comment:
Could we configure the `100ms` busy wait using the `instance` configuration? That would give us added flexibility to mess around with testing parameters should we want to.

Reply:
✅